e2e/test/scenarios/dashboard/x-rays.cy.spec.js

```diff
@@ -314,6 +309,11 @@ describe("scenarios > x-rays", { tags: "@slow" }, () => {
         },
       ],
     });
+
+    cy.findByTestId("viz-settings-button").click();
```

Comment:
This causes echarts to re-render which may cause tooltip to disappear prematurely.
Doing tooltip assertions first and moving this to the end of the test makes it not flaky anymore.